clients/bigquery/cast.go

```diff
@@ -59,6 +59,12 @@ func castColVal(colVal any, colKind columns.Column, additionalDateFmts []string)
 					colVal = fmt.Sprintf(`{"key":"%s"}`, constants.ToastUnavailableValuePlaceholder)
 				}
 			}
+
```

Comment:
Is the `colKind.KindDetails == typing.Struct` check above here redundant since we're switching on `colKind.KindDetails.Kind`?

Reply:
Yeah seems like it. I'll rm